packages/zapp/console/src/components/Executions/ExecutionDetails/Timeline/scaleContext.tsx

```diff
@@ -73,8 +73,8 @@ export const ScaleProvider = (props: ScaleProviderProps) => {
       newMax < MIN_SCALE_VALUE
         ? MIN_SCALE_VALUE
         : newMax > MAX_SCALE_VALUE
-        ? MAX_SCALE_VALUE
-        : newMax;
+          ? MAX_SCALE_VALUE
+          : newMax;
```

Comment:
I know this is existing code, but those ternaries are a but hard to comprehend 😜 maybe we can do something about it?